client/jetpack-connect/plans-grid.jsx

```diff
@@ -34,12 +34,11 @@ class JetpackPlansGrid extends Component {
 	renderConnectHeader() {
 		const { isLanding, translate } = this.props;
 
-		let headerText = translate( 'Your site is now connected!' );
-		let subheaderText = translate( "Now pick a plan that's right for you." );
+		const headerText = translate( 'Explore our Jetpack plans' );
+		let subheaderText = translate( "Now that you're connected, pick a plan that fits your needs." );
 
 		if ( isLanding ) {
-			headerText = translate( "Pick a plan that's right for you." );
-			subheaderText = '';
+			subheaderText = translate( 'Pick a plan that fits your needs.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Now that you're connected, pick a plan that fits your needs.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).